tests/test_jx/test_metadata.py

```diff
@@ -43,14 +43,14 @@ def test_meta(self):
         }
         self.utils.send_queries(pre_test)
 
-        test = set_default(test, {
+        test = {
             "query": {
                 "select": ["name", "table", "type", "nested_path"],
-                "from": "meta.columns",
-                "where": {"eq": {"table": table_name}}
```

Comment:
we must keep the `where` clause; it is the only clause we seem to use on the `meta.columns`, but it is  important.